examples/gnrc_networking/Makefile

```diff
@@ -32,8 +32,8 @@ USEMODULE += netstats_rpl
 # Optionally include DNS support. This includes resolution of names at an
 # upstream DNS server and the handling of RDNSS options in Router Advertisements
 # to auto-configure that upstream DNS server.
-# USEMODULE += sock_dns              # include DNS client
-# USEMODULE += gnrc_ipv6_nib_dns     # include RDNSS option handling
+USEMODULE += sock_dns              # include DNS client
+USEMODULE += gnrc_ipv6_nib_dns     # include RDNSS option handling
 
```

Comment:
You might also want to enable the `dns_cache` module - otherwise you'll make a new DNS request each time you resolve an address.

Might be worth adding a `dns_default` pseudo-module that includes all that, so a user only has to enable a single module for full DNS support.